docs/pages/repo/docs/handbook/deploying-with-docker.mdx

```diff
@@ -112,44 +112,39 @@ Our detailed [`with-docker` example](https://github.com/vercel/turbo/tree/main/e
 
 ```docker
 FROM node:alpine AS builder
+RUN apk add --no-cache libc6-compat
 RUN apk update
 # Set working directory
 WORKDIR /app
 RUN yarn global add turbo
 COPY . .
-RUN turbo prune --scope=web --docker
+RUN turbo prune --scope=api --docker
 
 # Add lockfile and package.json's of isolated subworkspace
 FROM node:alpine AS installer
+RUN apk add --no-cache libc6-compat
 RUN apk update
 WORKDIR /app
 
-# First install the dependencies (as they change less often)
+# First install dependencies (as they change less often)
 COPY .gitignore .gitignore
 COPY --from=builder /app/out/json/ .
 COPY --from=builder /app/out/yarn.lock ./yarn.lock
 RUN yarn install
 
-# Build the project
+# Build the project and its dependencies
 COPY --from=builder /app/out/full/ .
 COPY turbo.json turbo.json
-RUN yarn turbo run build --filter=web...
+RUN yarn turbo run build --filter=api...
 
 FROM node:alpine AS runner
 WORKDIR /app
 
 # Don't run production as root
-RUN addgroup --system --gid 1001 nodejs
-RUN adduser --system --uid 1001 nextjs
-USER nextjs
+RUN addgroup --system --gid 1001 expressjs
+RUN adduser --system --uid 1001 expressjs
+USER expressjs
```

Comment:
this should be USER nextjs, no?

Reply:
It seems I copied over the wrong Dockerfile, I added a comment to the api Dockerfile that tells people to copy the web Dockerfile over.